falco.yaml

```diff
@@ -544,3 +544,91 @@ metadata_download:
 base_syscalls:
   repair: false
   custom_set: []
+
+# metrics: periodic stats and resource utilization metrics, initial release
+#
+# --- [Description]
+#
+# `metrics` reflects a stats/metrics re-design from the ground up. Falco natively supports
+# resource utilization metrics and enhanced specialized metrics to monitor Falco's
+# performance in production. Metrics are exposed as monotonic counters or snapshots
+# emitted at a pre-defined interval. Each metric is captured in the same consolidated log message.
+# In addition, relevant wrapper fields are added, allowing the end user to perform customized
+# statistical analyses and correlations. Lastly, the metrics framework can be easily extended in the future.
+#
+# Notice: Schema and output field names are not guaranteed to be stable
+# and might change until `metrics` reaches a stable release.
+#
+# --- [Usage]
+#
+# Disabled by default.
+#
+# Env variable `-E FALCO_HOSTNAME=myhostname` customizes hostname for Kubernetes
```

Comment:
Not sure if I understand where this is applicable / useful (in other words, why would I set something like this?). Since this is an env var set outside of `falco.yaml`, should this be moved above or below the settings configured in this file?

Reply:
will add more info around it and maybe for here just saying you can set an environment variable is enough as there are multiple ways, for example you could do `sudo -FALCO_HOSTNAME=myhostname -E FALCO_BPF_PROBE=/root/.falco/falco-bpf.o ./userspace/falco/falco` ... and as said Linux and bash offer many ways.